README.md

```diff
@@ -3,7 +3,7 @@
 # adcc: Seamlessly connect your program to ADC
 | **Documentation**                                       | **Build Status**                                       |  **Installation**                                                                      |
 |:------------------------------------------------------- |:------------------------------------------------------ |:-------------------------------------------------------------------------------------- |
-| [![][docs-img]][docs-url] [![][binder-img]][binder-url] | [![][travis-img]][travis-url] [![][cov-img]][cov-url]  | [![][pypi-img]][pypi-url] [![][conda-img]][conda-url] [![][license-img]][license-url]  |
+| [![][docs-img]][docs-url] [![][binder-img]][binder-url] | [![][travis-img]][travis-url] [![][cov-img]][cov-url] [![Language grade: Python][lgtm-img]][lgtm-url]  | [![][pypi-img]][pypi-url] [![][conda-img]][conda-url] [![][license-img]][license-url]  |
```

Comment:
I would not put this badge in the table. It's not so important and it makes the table two-row, which is much harder to read. I'd add it somewhere else in the readme.